proto/service_user.proto

```
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+import "google/api/annotations.proto";
```

Buf (1.47.2)

import "google/api/annotations.proto": file does not exist

(COMPILE)